ui/app/components/clients/current.js

```diff
@@ -11,9 +12,20 @@ export default class Current extends Component {
   });
   @tracked selectedNamespace = null;
 
-  // TODO CMB get from model
   get upgradeDate() {
-    return this.args.upgradeDate || null;
+    let keyInfoObject = this.args.model.versionHistory.key_info;
```

Comment:
i think this will work if `key_info` is camel cased? (here and in the model)

Reply:
(Putting here for my brain)
sample response: 
```{
  "keys": [
    "1.9.0",
    "1.9.1",
    "1.9.2"
  ],
  "key_info": {
    "1.9.0": {
      "previous_version": null,
      "timestamp_installed": "2021-11-18T10:23:16Z"
    },
    "1.9.1": {
      "previous_version": "1.9.0",
      "timestamp_installed": "2021-12-13T11:09:52Z"
    },
    "1.9.2": {
      "previous_version": "1.9.1",
      "timestamp_installed": "2021-12-23T10:56:37Z"
    }
  }
}

Reply:
ah, so camel case in the model and then camelCase here?